ipn/ipnlocal/local.go

```diff
@@ -4441,7 +4440,7 @@ func (b *LocalBackend) setServeProxyHandlersLocked() {
 		backend := key.(string)
 		if !backends[backend] {
 			b.logf("serve: closing idle connections to %s", backend)
-			value.(*httputil.ReverseProxy).Transport.(*http.Transport).CloseIdleConnections()
+			value.(*reverseProxy).closeIdleConnections()
```

Comment:
change docs on `b.serveProxyHandlers` to say it points to `*reverseProxy`

Reply:
also swap the ordering. i think we want to first delete then close

Reply:
> first delete then close

why?

Reply:
its a race condition otherwise no?